README.md

```diff
@@ -18,9 +39,11 @@ Projekt zostanie zaimplementowany w następującej architekturze mikroserwisowej
 W razie potrzeby można zmniejszyć/zwiększyć ilość mikroserwisów.
 
 ## Technologie:
--	Frontend: Vue.js
--	Backend: Go
--	Database: MongoDB  
+-	Frontend: Vue.js 3.5
+-	Backend: Go 1.23
+-	Database: MongoDB 7.0 
+-   Containerization: Docker 27.3
```

Comment:
Jak już mamy całe Readme po polsku, to w technologiach też by się przydało zmienić na Baza danych i Konteneryzacja